src/apps/vpn/mozillavpn.cpp

```diff
@@ -197,15 +198,20 @@ MozillaVPN::MozillaVPN() : App(nullptr), m_private(new MozillaVPNPrivate()) {
   if (!Feature::get(Feature::Feature_webPurchase)->isSupported()) {
     ProductsHandler::createInstance();
   }
+
+#if defined(MZ_ANDROID) || defined(MZ_IOS)
+  PurchaseHandler* purchaseHandler = PurchaseIAPHandler::createInstance();
+  connect(purchaseHandler, &PurchaseIAPHandler::subscriptionCompleted, this,
+          &MozillaVPN::subscriptionCompleted);
+#else
   PurchaseHandler* purchaseHandler = PurchaseHandler::createInstance();
+#endif
   connect(purchaseHandler, &PurchaseHandler::subscriptionStarted, this,
           &MozillaVPN::subscriptionStarted);
   connect(purchaseHandler, &PurchaseHandler::subscriptionFailed, this,
           &MozillaVPN::subscriptionFailed);
   connect(purchaseHandler, &PurchaseHandler::subscriptionCanceled, this,
           &MozillaVPN::subscriptionCanceled);
-  connect(purchaseHandler, &PurchaseHandler::subscriptionCompleted, this,
```

Comment:
I don't understand how `subscriptionCompleted` arrives to `MozillaVPN` removing this line.

Reply:
See longer comment below, but in this approach (which I'm very open to changing) we do not want the web purchase to trigger this.